Library/Homebrew/cmd/audit.rb

```diff
@@ -193,6 +194,9 @@ def audit_file
     if present.include?("head") && present.include?("head block")
       problem "Should not have both `head` and `head do`"
     end
+    if present.include?("bottle modifier") && present.include?("bottle block")
+      problem "Should not have a bottle modifier & bottle block"
```

Comment:
Probably want to have consistent format with <code>Should not have both \`head\` and \`head do\`</code> in the above